examples/common/pigweed/protos/fabric_bridge_service.options

```diff
@@ -0,0 +1,6 @@
+chip.rpc.SynchronizedDevice.unique_id max_size:33
```

Comment:
uniqueid can be uptto 64 bytes I thought. Some of these sizes seems to be incorrect

Reply:
Basic information cluster only has 32 in my copy of the spec:

![image](https://github.com/user-attachments/assets/cd682080-484b-47c2-8fef-7aea4d753c2f)